src/tests/ftest/util/general_utils.py

```diff
@@ -773,24 +773,48 @@ def check_pool_files(log, hosts, uuid):
     return status
 
 
-def convert_list(value, separator=","):
-    """Convert a list into a separator-separated string of its items.
+def list_to_str(value, joiner=","):
+    """Convert a list to a string by joining its items.
+
+    Args:
+        value (list): list to convert to a string
+        joiner (str, optional): _description_. Defaults to ",".
```

Comment:
Well, don't keep us in suspense, what is the description?

I assume you're using a tool to pre-populate these annotations, can you share the details please?

Reply:
@ashleypittman Okay if I update this description in a followup PR? I need to rebase one of my PRs on these changes

Reply:
autoDocstring for VS Code: https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

Reply:
That's fine.  Could you do a quick "git grep _description_" and see if you can fill in any of the others as well please.

Reply:
Sure, I'll search all ftest for `_description_`, etc.

Reply:
FYI #11607